Vision/src/V1/Gapic/ImageAnnotatorGapicClient.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * Copyright 2017 Google LLC
+ * Copyright 2018 Google LLC
```

Comment:
Not sure if the Gapic generator should be handling this or if we want synthtool to do the gsubs.